scripts/reparse_flagged_rooms.py

```
@@ -0,0 +1,213 @@
+    failed = 0
+    failures: List[Dict] = []
+
+    for idx, business_id in enumerate(business_ids, start=1):
+        try:
+            logger.info("Reparsing %s/%s business_id=%s", idx, len(business_ids), business_id)
+            await service.collect_by_id(business_id)
+            success += 1
+        except Exception as e:
+            failed += 1
+            failures.append({"business_id": business_id, "reason": str(e)})
+            logger.exception("Failed to reparse business_id=%s: %s", business_id, e)
```

Ruff (0.15.2)

[warning] Redundant exception object included in `logging.exception` call

(TRY401)